template.rb

```diff
@@ -183,7 +183,7 @@ def cleanup_package_json
   package_json["engines"] = build_engines_field
 
   # ensure that all dependency constraints are normalized
-  %w[dependencies devDependencies].each { |k| package_json[k] = normalize_dependency_constraints(package_json[k]) }
+  # %w[dependencies devDependencies].each { |k| package_json[k] = normalize_dependency_constraints(package_json[k]) }
```

Comment:
Normalizing the deps to always start with `^` breaks the exact dep we need currently need to specify for bootstrap

Reply:
@G-Rath This is just roughly commented out but I'm thinking maybe doing dependency normalisation might cause more problems than it solves?